### ISSUE TITLE
[WCAG] Ensure elements use unique IDs

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3220]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Element IDs should be prefixed with the mapID to ensure they are unique.

Update: This is linked to #3221 which stipulates this note:
_NOTE: We should have the class-name-${mapId} format -> Not only we need a wrapper to 'search in the DOM' but a wrapper to 'create those ids' as well so we always create the id in the 'prefix-mapId' format._

**Comment by kenchase** [comment]
Closed as part of #3629
